src/styles/header.scss

```diff
@@ -0,0 +1,71 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0 auto;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  transition: all 300ms;
+
+  &__logo-img {
+    height: $logo-img-size;
+    width: $logo-img-size;
```

Comment:
Ensure that the variable `$logo-img-size` is defined in your SCSS files. If it's not defined, it will cause an error during compilation.